### PR TITLE
no usage of global.window check, because files are only run in a cont…

### PR DESCRIPTION
**Pull request by ScottDowne**
…ext in which they can run successfully

### CODE REVIEW

**Comment by ScottDowne** [comment]
Adding comment with explaining this better very soon...

**Comment by ScottDowne** [comment]
Experimenting, bear with me.

**Comment by ScottDowne** [comment]
The idea I am experimenting with is this:

Client side only:
public/js/*
Client.jsx
componentDidMount

Universal:
scripts/*
jsx files minus Client.jsx

Server side only:
Everything else

**Comment by ScottDowne** [comment]
Actually, looks like whatever is in scripts now is very different than what it was. It's basically server side only now plus some simple require json data files.

That makes things even easier!

**Comment by ScottDowne** [comment]
@alicoding Ha, I want you to try your hardest to shoot this down. I'll accept nothing less.

File: components/client.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import assign from 'react/lib/Object.assign';
 import Router from 'react-router';
 import currencies from '../scripts/currencies.js';
 import routes from './routes.jsx';
-import {intlDataFor, intlData} from '../scripts/i18n';
+import {intlDataFor, intlData} from '../public/js/i18n.js';
```
Comment (alicoding) [inline components/client.jsx]: Unnecessary .js here but just nit picking lol

File: components/routes.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Route } from 'react-router';
-import pages from '../scripts/pages.js';
+import pages from '../data/pages.js';
```
Comment (alicoding) [inline components/routes.jsx]: I like that we abstract things to its own descriptive directory here

File: locales/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+function getLocales(arr) {
+  return arr.map(function(item) {
+    return item.replace("./", "").replace(".json", "");
+  });
+}
+
+var reqF = require('enhanced-require');
+var req = reqF(module);
```
Comment (alicoding) [inline locales/index.js]: Not sure why I didn't thought of this lol! It can pretty much run both client and server with this module.

**Comment by alicoding** [comment]
I still don't know what makes your changes work here, but kinda get the idea I guess... As long as this is working I'm happier than having to use that global.window, so r+

**Pull request merged**